Anlab.Mvc/Client/components/OrderForm.tsx

```diff
@@ -203,18 +203,20 @@ export default class OrderForm extends React.Component<undefined, IOrderState> {
 
                     <SampleTypeSelection sampleType={sampleType} onSampleSelected={this.onSampleSelected} />
 
-                    <div className="form_wrap">
+                    <div className="form_wrap" id="quantity">
                         <label className="form_header">How many samples will you require?</label>
                         <Quantity quantity={quantity} onQuantityChanged={this.onQuantityChanged} />
                     </div>
                     <AdditionalEmails addedEmails={additionalEmails} onEmailAdded={this.onEmailAdded} onDeleteEmail={this.onDeleteEmail}/>
-                    <Project project={project} handleChange={this.handleChange} />
+                    <div id="project">

```

Comment:
@laholstege this one is ok unless you feel like adding a label describing what project is (which will be good eventually)